traffic-id.yaml

```diff
@@ -126,6 +157,84 @@ tls-sni-patterns:
       - instagram.com
       - cdninstagram.com
 
+  - id: feedly
+    labels:
+      - news
+      - rss
+    patterns:
+      - feedly.com
+
+  - id: nos
+    labels:
+      - news
+    patterns:
+      - nos.nl
+
+  - id: chartbeat
+    labels:
+      - stats
+      - tracking
+    patterns:
+      - chartbeat.net
+
+  - id: github
+    labels:
+      - dev
+    patterns:
+      - github.com
+
+  - id: github-tracking
+    labels:
+      - dev
+      - stats
+      - tracking
+    patterns:
+      - collector.githubapp.com
+
+  - id: mozilla
+    labels:
+      - browser
+    patterns:
+      - firefoxusercontent.com
+      - services.mozilla.com
+
+  - id: rememberthemilk
+    labels:
+      - todo
```

Comment:
I wonder if this is better labeled task-management or something. There is overlapping functionality with tools like Trello, Asana and Omnifocus, which are way more than "todo" apps, but close enough that I'd like them all labelled together.